feature_layers/feature-layer-query/src/main/java/com/esri/samples/feature_layer_query/FeatureLayerQuerySample.java

```diff
@@ -148,16 +153,16 @@ public void start(Stage stage) {
       // set renderer for feature layer
       featureLayer.setRenderer(new SimpleRenderer(fillSymbol));
 
-      // create a ArcGISMap with basemap topographic
-      final ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
-
-      // add feature layer to operational layers
-      map.getOperationalLayers().add(featureLayer);
+      // create a map with a basemap style
+      final ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
 
-      // create a view for this ArcGISMap
+      // create a map view and set its map
```

Comment:
a few other instances in this PR of where this comment could be slightly improved, I won't comment again to save notifications! No worries as with the basemap comment changes if you don't have bandwidth for it this time round, it's not urgent. 
```suggestion
      // create a map view and set the map to it
```